app/uk/gov/hmrc/gform/gform/NewFormController.scala

```diff
@@ -18,7 +18,6 @@ package uk.gov.hmrc.gform.gform
 
```

Comment:
reverting this class to the previous state. All logic for clearing session is moved to EmailAuthSessionPurgeFilter.scala